app/assets/stylesheets/logixRelease.scss

```diff
@@ -242,6 +242,11 @@ a.btn.btn-outline-danger {
   border-radius: 8px;
   padding: 4px;
   box-shadow: 3px 3px 6px $shadow-black;
+  transition: box-shadow .2s ease-in;
+}
+
+.patreon:hover {
```

Comment:
Move to about.scss, also prefer sass pseudo class syntax

Reply:
This is not in about page, this is in contribute page

Reply:
then contribute.scss

Reply:
👍